viper.go

```diff
@@ -908,8 +910,11 @@ func (v *Viper) InConfig(key string) bool {
 func SetDefault(key string, value interface{}) { v.SetDefault(key, value) }
 func (v *Viper) SetDefault(key string, value interface{}) {
 	// If alias passed in, then set the proper default
-	key = v.realKey(strings.ToLower(key))
-	v.defaults[key] = value
+	flat := bellows.FlattenPrefixed(value, key)
```

Comment:
We flatten the `value` if it is Struct or Map.  Therefore, if we have [a Config defined as a struct](https://github.com/doublerebel/fabio/blob/5a05baa4418abdb74c8cbc39d465858b18f0e1db/config/config.go) and [define defaults as an instance of the Config struct](https://github.com/doublerebel/fabio/blob/5a05baa4418abdb74c8cbc39d465858b18f0e1db/config/default.go), then we can [set any or all defaults in a single call](https://github.com/doublerebel/fabio/blob/5a05baa4418abdb74c8cbc39d465858b18f0e1db/config/load.go#L45).